### ISSUE TITLE
Improve the alignment of morphology pop-up for present infinitive verbs

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#429]
lookup (or double click from https://www.thelatinlibrary.com/lucretius/lucretius2.shtml) videre.
see the morphology pop-up   - can we improve the alignment? this is also in the release.

![Screen Shot 2020-06-08 at 2 03 15 PM](https://user-images.githubusercontent.com/41396793/84030067-af074f80-a993-11ea-915f-36c6d85b7dfb.png)

**Comment by balmas** [comment]
@kirlat can you take a look at this?

**Comment by kirlat** [comment]
Sure, I will!

**Comment by kirlat** [comment]
Do I understand correctly that here we need to shift `3rd person plur.` to the left so that it will be below the `2nd person sing.`?

**Comment by balmas** [comment]
yes I believe that's the request, correct @monzug ?

**Comment by monzug** [comment]
how about this:
pres. infin. act
2nd person sing. pres. ind. pass
2nd person sing. pres. imp. pass
3rd person plur. perf. ind. act

see also amare (or vivere) for similar pop-up

**Comment by kirlat** [comment]
From technical standpoint, what @monzug suggest is doable. Moving items to the right to align them vertically (one below the other) is much more complex. We have several nested loops there, and we need to combine it with special placement of special items on special occasions. We probably would need to take into consideration language and other lemma attributes. That could brake too easily with some morph output that is not-so-standard.

**Comment by balmas** [comment]
I don't want to go into the guts of that code right now. So if there is a simple low-risk way to make it look better, I would be okay with that, but nothing risky.

**Comment by monzug** [comment]
agree. it has been there for ever

**Comment by balmas** [comment]
moving out of the release. Should be addressed together with refactoring of grouping needed for #101 and other similar items.

**Comment by balmas** [comment]
>From technical standpoint, what @monzug suggest is doable. Moving items to the right to align them vertically (one below the other) is much more complex. We have several nested loops there, and we need to combine it with special placement of special items on special occasions. We probably would need to take into consideration language and other lemma attributes. That could brake too easily with some morph output that is not-so-standard.

Let's do the easy solution (as in https://github.com/alpheios-project/alpheios-core/issues/429#issuecomment-640685444). Please make the fix in a branch off of the incr-3.3.x branch.

**Comment by kirlat** [comment]
I've made the fix, the output for `videre` looks like below now:
![image](https://user-images.githubusercontent.com/18631055/105827449-73310100-5fdb-11eb-81a4-ebe0b3430210.png)

The change, however, affects all other words that have inflections. Inflections are moved to the next line as shown in the example below:

before:
![image](https://user-images.githubusercontent.com/18631055/105827835-e76ba480-5fdb-11eb-8b91-f62c66de13bb.png)

after:
![image](https://user-images.githubusercontent.com/18631055/105827998-1550e900-5fdc-11eb-9fb5-a5f2e5d6bfad.png)

Is that change acceptable? Do you know of any other words that produces the output this solution should be tested with?

**Comment by monzug** [comment]
I found videre in https://texts.alpheios.net/text/urn:cts:latinLit:phi0620.phi001.alpheios-text-lat1/passage/1.1, then in the same passage we have conquirere, perfundere, procedere

![Screen Shot 2021-01-26 at 3 29 24 PM](https://user-images.githubusercontent.com/41396793/105858380-8ba91780-5feb-11eb-9655-badebe09d942.png)
![Screen Shot 2021-01-26 at 3 28 54 PM](https://user-images.githubusercontent.com/41396793/105858384-8c41ae00-5feb-11eb-9d50-1133a0ba49ad.png)

**Comment by monzug** [comment]
and amare from lookup

![Screen Shot 2021-01-26 at 3 58 43 PM](https://user-images.githubusercontent.com/41396793/105861907-7e8e2780-5fef-11eb-82a1-d3401cbf565d.png)

**Comment by balmas** [comment]
> Honestly I do not like the way the change impacts all words. we prefer to have a more compact pop-up, specially for mobile.

Agree. @kirlat if this is all we can do without risky changes, let's just drop it for now. Thanks.

**Comment by kirlat** [comment]
The more compact solution is possible only if we change the structure of the inflection group display elements, maybe even across several Vue components, and I'm afraid to think about what implications for the multitude of all the test cases this may cause. So I would say we'd better delay it, if we can afford to do so. In the end, this probably affects only a relatively small subgroup of all the words, but the changes we're required to make to fix it would affect all the words in all the languages.

**Comment by balmas** [comment]
ok. Moving this out of the pre-pause project then. thanks.
